### DISCUSSION TITLE
Tab focus breaking keyboard ⌘ back control

### DISCUSSION THREAD

**Discussion by psugihara** [radix-ui/primitives#2528] [Help]
When I interact with Tabs, it breaks my hotkey for going back. Is this intended behavior? If so, is there an easy way to disable keyboard controls? Before isolating it to the tab control, it made me think that the back button had broken.

**Comment by benoitgrelard** [comment]
What is the issue exactly? We only control arrow keys. I am able to navigate back…

**Comment by psugihara** [comment]
To go back from a website, I use the ⌘ ← hotkey. It is unexpected that the arrow keys would be intercepted by the tab. For example, if I click "Pull requests" above, I can then do ⌘ ← to go back to this discussion page. If you feel strongly about the arrow key control, it would be nice if you ignored it when a modifier key (command, control, shift) is also pressed.

**Reply by benoitgrelard** [reply]
oh I see the issue now! I use a different shortcut myself to navigate back so didn't see it initially.
Yes it does look like a bug. I will convert the issue and raise a fix.

**Reply by psugihara** [reply]
thanks!

**Comment by benoitgrelard** [comment]
#2732
